docs/rules.md

```diff
@@ -3,6 +3,7 @@
 * [attribute-indentation](rule/attribute-indentation.md)
 * [block-indentation](rule/block-indentation.md)
 * [eol-last](rule/eol-last.md)
+* [element-attributes-order](rule/element-attributes-order.md)
```

Comment:
This file has been deleted so you will want to rebase on the latest master and run `yarn update` to update the README instead.